### ISSUE TITLE
Adding theme change setting to options.html

### ISSUE THREAD

**Issue by itsmais** [itsmais/french-accents#1]
### Request details
The current layout uses the following color palette:
```
Background: #BCECE0 
Text: #36EEE0
Shift: #F652A0
Button: #4C5270
```
It would be nice to have, in the `options.html` page, an option for the user to pick a different palette.
The number of palette choices has to be between 4 - 8  palettes (you can use anything from https://colorhunt.co/).

### Submitting a pull request
Please link this issue to the pull request. I will review it within 24 hours. Thank you!

**Comment by raji2004** [comment]
i would like to be assigned this issue

**Comment by itsmais** [comment]
great, I will await your PR @raji2004 
thanks!

**Comment by raji2004** [comment]
there is nothing in the options.html

**Comment by itsmais** [comment]
@raji2004 yes, that's why we're adding this.
let's not worry about the styles, just the functionality for now.

**Comment by raji2004** [comment]
Oh so u want a way to switch background colour of options.html?

**Comment by itsmais** [comment]
Yes @raji2004 , the entire set of colors mentioned in my first comment.

**Comment by itsmais** [comment]
@raji2004 If you need help with that, I find this [Getting Started example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/tutorials/getting-started) very useful. Let me know if you have any questions.

**Comment by itsmais** [comment]
This issue is open for other contributors now.

**Comment by itsmais** [comment]
resolved in https://github.com/itsmais/french-accents/commit/29ee2e506a5c154102b32dcb234160f450167e0c
